VideoLocker/res/layout/panel_custom_action_bar.xml

```diff
@@ -18,13 +18,13 @@
         android:layout_marginLeft="11dp"
         android:layout_marginStart="11dp"
         android:orientation="vertical"
+        android:contentDescription="@string/cancel_btn"
```

Comment:
Just confirmed with Ehmad, there's no accessibility label for cross button on this screen:

![device-2015-08-21-123108](https://cloud.githubusercontent.com/assets/1710804/9407293/49cb2958-4821-11e5-9ae1-5af37845d586.png)

VoiceOver reads it as "ic cancel button" which I misunderstood to be "I cancel button"